vector/src/main/java/im/vector/app/features/home/HomeDetailViewEvents.kt

```diff
@@ -28,5 +28,6 @@ sealed class HomeDetailViewEvents : VectorViewEvents {
     object InviteNoTchapUserByEmail : HomeDetailViewEvents()
     data class GetPlatform(val email: String) : HomeDetailViewEvents()
     data class OpenDirectChat(val roomId: String) : HomeDetailViewEvents()
+    data class PromptCreateDirectChat(val userId: String) : HomeDetailViewEvents()
```

Comment:
I would prefer to have here a User as param instead of a userID in order to keep user info (avatar and display name). These information may be available fropm the contacts list where the user (contact) has been selected

Reply:
the origin of the change will take place [here](https://github.com/tchapgouv/tchap-android-v2/pull/163/files#diff-4618209df1973cd9eabeb81eef8adc6a6449806e419fef682f77dd1b629448ecL398)